XRTK-Core/Packages/com.xrtk.core/Interfaces/Providers/Controllers/IMixedRealityHand.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using XRTK.Definitions.Utilities;
+
+namespace XRTK.Interfaces.Providers.Controllers
+{
+    /// <summary>
+    /// Hand definition, used to provide access to hand joints and other data.
+    /// </summary>
+    public interface IMixedRealityHand : IMixedRealityController
+    {
+        /// <summary>
+        /// Get the current pose of a hand joint.
+        /// </summary>
+        /// <remarks>
+        /// Hand bones should be oriented along the Z-axis, with the Y-axis indicating the "up" direction,
+        /// i.e. joints rotate primarily around the X-axis.
+        /// </remarks>
+        bool TryGetJoint(TrackedHandJoint joint, out MixedRealityPose pose);
```

Comment:
This is a good way to go, but its inconsistent with IMixedRealityHandJointService

Reply:
Then we need to build on that towards a more common standard